### PR TITLE
HOLD: Update cf-typography docs with italic web font removal + font-display: fallback

### PR DESCRIPTION
**Pull request by cfarm**
Updates the docs after removal of Avenir Next Italic in #714 and addition of font-display font-face property in #691.

Also includes whatever other changes from latest release might affect CSS.

### CODE REVIEW

File: _includes/usage/cf-typography.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ project by duplicating the variable `@key: value`.
 
 ```
 @webfont-regular: 'AvenirNextLTW01-Regular';
-@webfont-italic: 'AvenirNextLTW01-Italic';
```
Comment (Scotchester) [inline _includes/usage/cf-typography.md]: I think it's worth adding a note to the docs (above or below here) about _why_ we're doing this, noting the fact that (a) faux italic Avenir Next Regular is almost indistinguishable from Avenir Next Italic, and (b) we don't use a lot of italic as a matter of course in our websites.

Comment (anselmbradford) [reply _includes/usage/cf-typography.md]: Might roll it into https://github.com/cfpb/capital-framework/issues/700 if that makes more sense.

Comment (Scotchester) [reply _includes/usage/cf-typography.md]: Oy, just realized that this is the docs PR and it needs to be updated in the main branch's usage.md file first. Stand by.

**Comment by Scotchester** [comment]
These changes got merged in via other means.

**Pull request closed without merging**